widget/ui/src/components/Alert/Alert.types.ts

```diff
@@ -8,5 +9,8 @@ export interface PropTypes {
   footer?: ReactNode;
   action?: ReactNode;
   variant?: 'alarm' | 'regular';
-  containerStyles?: CSSProperties;
+  containerStyles?: { [key: string]: string };
+  titleContainerStyles?: { [key: string]: string };
```

Comment:
Please remove this. this only using to handle `gap` style. you can skip this, it's a design mistake. we shouldn't handle.

Reply:
Removed